### PR TITLE
fixes faec

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Jeez, Fethas.

https://github.com/ParadiseSS13/Paradise/pull/7281 broke examines; this fixes that.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -443,7 +443,7 @@
 		msg += "<span class='warning'><b>[src] has \a [implant] sticking out of [t_his] flesh!</b></span>\n"
 	if(digitalcamo)
 		msg += "[t_He] [t_is] repulsively uncanny!\n"
-	if(!(skipface || wear_mask.flags_inv & HIDEFACE || wear_mask.flags_cover & MASKCOVERSMOUTH) && is_thrall(src) && in_range(user,src))
+	if(!(skipface || wear_mask.flags_inv & HIDEFACE || (wear_mask && wear_mask.flags_cover & MASKCOVERSMOUTH)) && is_thrall(src) && in_range(user,src))
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/examine.dm]: Don't forget `wear_mask.flags_inv & HIDEFACE` this one.

Comment (FlattestGuitar) [inline code/modules/mob/living/carbon/human/examine.dm]: Oh dear!

**Comment by KasparoVy** [comment]
Oh god why are there spaces between the inside brackets but not the outside

**Comment by Fox-McCloud** [comment]
For future reference, though it's not *required*, when you make a fix PR that fixes something so common and so high priority, it's probably best to make SOME changelog so players know when it's been fixed.

**Pull request merged**